google-cloud-firestore/src/main/java/com/google/cloud/firestore/UpdateBuilder.java

```diff
@@ -723,40 +735,88 @@ BatchState getState() {
     return state;
   }
 
-  Set<DocumentReference> getDocuments() {
-    return documents;
+  boolean hasPath(String path) {
```

Comment:
This should also take a DocumentReference and likely needs to be renamed accordingly.